src/parser/instruction.rs

```diff
@@ -466,6 +466,33 @@ mod tests {
         }]
     );
 
+    #[test]
+    fn parse_set_phase() {
+        let tokens = lex(r#"SET-PHASE 0 "rf" 1.0; SET-PHASE 0 1 "rf" theta"#);
+        let (remainder, parsed) = parse_instructions(&tokens).unwrap();
+        let expected = vec![
+            Instruction::SetPhase {
+                frame: FrameIdentifier {
+                    name: String::from("rf"),
```

Comment:
Ah yes, the ol' "how to create a `String` from a `str`" conversation 😆. The main options are, I believe:

1. `String::from("str")`
2. `"str".to_owned()`
3. `"str".to_string()`

I'm definitely guilty of switching between them a bit, but I prefer the first one because it's more obvious when skimming that you've just allocated some new memory on the heap- which makes it easier to make tiny optimizations later. [The book](https://doc.rust-lang.org/stable/book/ch04-01-what-is-ownership.html#the-string-type), [Rust By Example](https://doc.rust-lang.org/stable/rust-by-example/std/str.html#strings), [the `String` docs](https://doc.rust-lang.org/stable/std/string/struct.String.html), and [the `str` docs](https://doc.rust-lang.org/std/primitive.str.html#method.get) all default to `String::from`, so that seems to be the recommended method.

Reply:
I created an internal Runbook issue to track adding to our internal style guide.